_posts/static/2017-12-08-getting-started.md

```diff
@@ -11,24 +11,26 @@ The Flyve MDM plugin for GLPI integrates the intuitive and outstanding features
 
 As GLPI is a Free Asset and IT Management Software package, we wanted to provide a plugin that helps you to keep control of your mobile devices, here we'll guide you in the steps to manage your mobile fleet from the basics.
 
-## Adding agents
+## Inviting Agents
 
-In order to add agents you must invite them, go to the Invitations section, click on the "+" button and write the email of the user whose device you'll control.
+In order to add the devices to your fleet, you must invite them first by sending an email to the owner/user of the device.
```

Comment:
and here I would say "you must invite the owner of the device to enroll by sending an email" because we don't invite a device: only the user.